posts/2020-05-15-five-years-of-rust.md

```diff
@@ -0,0 +1,508 @@
+---
+layout: post
+title: "Five Years of Rust"
+author: The Rust Core Team
+---
+
+With all that's going on in the world you'd be forgiven for forgetting that as
+of today, it has been five years since we released 1.0 in 2015! Rust has changed
+a lot these past five years, so we wanted reflect back on all of our
+contributors' work over these past five years.
+
+**If this is the first time you're reading about Rust:** Rust is a general
```

Comment:
Why does it feel weird to you? I'm not sure that the comparison to release posts is fair.  I feel  (and hope) that this post has much a broader audience than the typical release posts. I'm more trying to explain and give context around the organisation and community that has grown project more than the actual software itself.

The choice to put emphasis, is so that if you're someone who's never heard of Rust and you land on this page, you see a short paragraph that stands out and is deliberately targeted at you, that (hopefully) gives enough context and makes you want to read the rest of the article, and if you're someone who's a regular reader or Rust user, you read that one sentence and you know can skip the rest of the paragraph and move on.

Without the emphasis I feel that the paragraph the background fades into looking like a second paragraph to the introduction, which it really isn't.

### With Emphasis
<img width="881" alt="Screenshot 2020-05-07 at 15 09 31" src="https://user-images.githubusercontent.com/4464295/81300440-f2f6f400-9077-11ea-8c5e-01a81d68485d.png">

### Without Emphasis
<img width="865" alt="Screenshot 2020-05-07 at 15 37 38" src="https://user-images.githubusercontent.com/4464295/81301019-c8f20180-9078-11ea-8d27-b2847c1845a4.png">

Reply:
I'm fine without bolding text.

Reply:
Oh, I should've explained myself better, sorry!

I'd remove the whole "If this is the first time you're reading about Rust:" text. I'm not exactly sure *why*, but it feels weird to me.